app/client/src/pages/Editor/GeneratePage/components/CrudInfoModal.tsx

```diff
@@ -89,27 +88,83 @@ const ImageWrapper = styled.div`
   justify-content: center;
 `;
 
-function Header() {
+const SuccessContentWrapper = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`;
+
+const SuccessImage = styled.img`
+  margin: ${(props) => props.theme.spaces[6]}px;
+`;
+
+const STEP = {
+  SHOW_SUCCESS_GIF: "show_success_gif",
+  SHOW_INFO: "show_info",
+};
+
+function InfoContent({
+  onClose,
+  successMessage,
+}: {
+  onClose: () => void;
+  successMessage: string;
+}) {
   return (
-    <HeaderContents>
-      <Heading> {GEN_CRUD_INFO_DIALOG_HEADING()}</Heading>
-    </HeaderContents>
+    <>
+      <Content>
+        <Text
+          className="info-subtitle"
+          dangerouslySetInnerHTML={{
+            __html: successMessage,
+          }}
+          type={TextType.P1}
+        />
+        <ImageWrapper>
+          <InfoImage alt="CRUD Info" src={getInfoImage()} />
+        </ImageWrapper>
+      </Content>
+
+      <ActionButtonWrapper>
+        <ActionButton
+          category={Category.primary}
+          onClick={() => {
+            onClose();
+          }}
+          size={Size.medium}
+          text="GOT IT"
+        />
+      </ActionButtonWrapper>
+    </>
   );
 }
-
 const getInfoImage = (): string =>
   `${S3_BUCKET_URL}/crud/working-flow-chart.png`;
 
 function GenCRUDSuccessModal(props: Props) {
-  const { crudInfoModalOpen } = props;
+  const { crudInfoModalOpen, generateCRUDSuccessInfo } = props;
 
   const dispatch = useDispatch();
+  const [step, setStep] = useState(STEP.SHOW_SUCCESS_GIF);
 
   const onClose = () => {
     AnalyticsUtil.logEvent("CLOSE_GEN_PAGE_INFO_MODAL");
-    dispatch(setCrudInfoModalOpen(false));
+    dispatch(setCrudInfoModalData({ open: false }));
   };
 
+  const successMessage =
+    (generateCRUDSuccessInfo && generateCRUDSuccessInfo.successMessage) ||
+    GEN_CRUD_INFO_DIALOG_SUBTITLE();
```

Comment:
Minor: Instead of using the message directly, we should use the function `createMessage()`
```suggestion
    createMessage(GEN_CRUD_INFO_DIALOG_SUBTITLE);
```

This helps us in i18n in the future.